harness/determined/cli/_color_json.py

```diff
@@ -0,0 +1,92 @@
+"""
+colorize JSON output.
+test string:
+   echo '{"a": "b", "c": ["1", 2.0, 3.1, 5, "4", {"d": true, "e": null, "f": [1, 2, 3]}]}'
+"""
+
+import json
+from typing import Any, TextIO
+
+from termcolor import colored
+
+KEY = "blue"
+PRIMITIVES = "white"
+SEPARATORS = "yellow"
```

Comment:
too much?

Reply:
ah I should have been more clear here I was pointing to what this actually does which is coloring the brackets and commas, something jq doesn't do by default and asking if that's too many colors.

Reply:
Ah, if it were me I'd stick to jq, which is tried and trusted, but I don't see it as a big deal either way.